Makefile

```diff
@@ -14,11 +14,23 @@ REGISTERS_TEX += sw_registers.tex
 REGISTERS_CHISEL += dm_registers.scala
 REGISTERS_CHISEL += abstract_commands.scala
 
+INCLUDES_TEX = introduction.tex
```

Comment:
I didn't because at times the current directory contains some autogenerated tex files as well. That shouldn't obviously lead to problems, but it felt more right to be explicit.

If you still prefer *.tex I'll change it.